docs/pages/access-controls/teleport-policy/policy-connections.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Teleport Policy Connections
+description: Connections in Access Graph with Teleport Policy.
+---
+
+## Connecting to resources
+
+### Graph paths
+
+Teleport Access Graph shows the relationships between users, roles, and
+resources. It does this by showing paths between nodes. Paths are the
+relationships between nodes. Paths always connect nodes in the following order:
+
+1. Users
+1. User Groups
+1. Actions
+1. Resource Groups
+1. Resources
+
+Paths can be divided into two categories:
+
+1. Allow paths
```

Comment:
I would indent the images/paragraphs below the ordered list so it renders more clearly.